code/modules/cargo/packs/armory.dm

```diff
@@ -193,11 +193,11 @@
 					/obj/item/clothing/head/helmet/rus_ushanka,
 					/obj/item/clothing/suit/armor/vest/russian_coat,
 					/obj/item/gun/ballistic/shotgun/boltaction,
-					/obj/item/gun/ballistic/shotgun/boltaction)
+					/obj/item/staff/broom) //Its... A real thing that was done in the Russian army
```

Comment:
Eeeh, broom brigades were not restricted to that country, you may want to modify the comment, or even remove this line cause people may not understand the reference.

Reply:
In the first Grate War, their was not a lot of guns and ammo.
Russians were forced to be in pares of two, one person would have a clip of ammo and a broom to make it look like a gun at  a dissidence. Well the other one had the gun.